### PR TITLE
feat(core): update prettier to v2.1.2

### PR DESCRIPTION
**Pull request by mehrad-rafigh**
## Current Behavior
prettier is at v2.0.4

## Expected Behavior
prettier is at v2.1.2

### CODE REVIEW

File: packages/workspace/src/migrations/update-10-3-0/update-10-3-0.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { formatFiles, updatePackagesInPackageJson } from '@nrwl/workspace';
+import { join } from 'path';
+import { chain } from '@angular-devkit/schematics';
+
+const updatePrettierPackage = updatePackagesInPackageJson(
```
Comment (vsavkin) [inline packages/workspace/src/migrations/update-10-3-0/update-10-3-0.ts:5]: Can you rename it to updatePackages? Other packages can be updated there as well

**Comment by vsavkin** [comment]
Thank you for your PR!

Could you also update prettier in the root package.json in the Nx repo itself?

**Review by vsavkin — CHANGES_REQUESTED** [review]

**Comment by mehrad-rafigh** [comment]
Hi @vsavkin 

> Could you also update prettier in the root package.json in the Nx repo itself?
I updated prettier to 2.1.2 in the package.json of the repo itself and renamed the rule to `updatePackages`

**Comment by mehrad-rafigh** [comment]
Hi @vsavkin Any chance this can get merged?

**Comment by vsavkin** [comment]
Sorry about the delay. Merged it here: https://github.com/nrwl/nx/commit/e3f318cde33b3dc68c0374af77271d66f6a01af4

Thank you!

**Pull request closed without merging**